Gemfile

```diff
@@ -3,4 +3,3 @@ gem "simplecov"
 gemspec
 
 #gem 'chef-provisioning', path: '../chef-provisioning'
-#gem 'chef-provisioning', github: 'chef/chef-provisioning', branch: 'master'
```

Comment:
Its okay to leave this in - its useful for local development to only need to uncomment this